src/components/SignUp/SignUpForm.tsx

```diff
@@ -18,59 +18,88 @@ function SignUpForm() {
 
 	const [popup, setPopup] = useState(false);
 	const [popupMsg, setPopupMsg] = useState('');
+
+	const PASSWORD_LENGTH = 6;
```

Comment:
I did note that we should as the client about the complexity but this is very good, thank you